app/src/main/java/com/internship/move/feature/onboarding/OnboardingItem.kt

```diff
@@ -0,0 +1,12 @@
+package com.internship.move.feature.onboarding
+
+import android.os.Parcelable
+import kotlinx.parcelize.Parcelize
+
+@Parcelize
+data class OnboardingItem(
```

Comment:
Just in case you forgot, you can use res values instead of strings, just like for the image, and use `@DrawableRes` and `@StringRes` as we discussed. This is only a suggestion of preference, it's up to you 😁